index.js

```diff
@@ -158,7 +158,7 @@ class FastifyOtelInstrumentation extends InstrumentationBase {
       instance.decorateRequest(kRequestSpan, null)
       instance.decorateRequest(kRequestContext, null)
 
-      instance.addHook('onRoute', function (routeOptions) {
+      instance.addHook('onRoute', function onRouteHook (routeOptions) {
```

Comment:
the name should be something meaningful.

In this scenario I would call it


```suggestion
      instance.addHook('onRoute', function otelWireRoute (routeOptions) {
```